core-bundle/contao/themes/flexible/styles/basic.css

```diff
@@ -1056,13 +1034,26 @@ a.tl_submit {
 	.split-button > button[type="button"]:focus {
 		outline: none;
 	}
+
+	.w16, .w25, .w33, .w50, .w66, .w75 {
+		--widget-cols: 2;
+	}
 }
 
 /* Handheld */
-@media (max-width: 767px) {
-	.w25, .w33, .w50, .w66, .w75 {
-		float: none;
-		width: calc(100% - 30px);
+@media (max-width: 767.98px) {
+
+	.widget-group {
+		grid-column-gap: 0;
```

Comment:
```suggestion
		column-gap: 0;
```

Reply:
Rewrote it to:
```css
.widget-group {
	--widget-col-amount: 12;
	--widget-gap-rows: 0;
	--widget-gap-cols: 0;
	display: grid;
	gap: var(--widget-gap-rows) var(--widget-gap-cols);
	grid-template-columns: repeat(var(--widget-col-amount), minmax(0, 1fr));
}
```

This also allows for modification of the row itself if people need it in their own backend widgets

Reply:
I updated the description based on that

#### Additional classes

Introcuded `.w16` on that occasion

___

#### Custom properties

Following custom properties were added:

| Custom property | Description |
| :- | :- |
| `--widget-col-amount` | Column amount of the backend-grid<br>_(Defaults to 12)_ |
| `--widget-cols` | Amount of space that this takes per row<br>_(Can be used to create more specific widget classes)_ |
| `--widget-gap-rows` | Row gap for widget-group container |
| `--widget-gap-cols` | Column gap for widget-group container |